src/extensibility/ExtensionManagerViewModel.js

```diff
@@ -292,6 +292,20 @@ define(function (require, exports, module) {
             });
     };
 
+    ExtensionManagerViewModel.prototype._setSortedExtensionList = function (extensions, isTheme) {
+        this.filterSet = this.sortedFullSet = registry_utils.sortRegistry(extensions, "registryInfo")
+            .filter(function (entry) {
```

Comment:
There seems to be some redundancy here, seems like same could be achieved with:

``` js
return entry.registryInfo && entry.registryInfo.metadata;
```

Do correct me if I am wrong 👍